jenkins/job-configs/kubernetes-jenkins/kubernetes-garbagecollector.yaml

```diff
@@ -72,5 +72,115 @@
                 export KUBEMARK_NUM_NODES="100"
                 # The kubemark scripts build a Docker image
                 export JENKINS_ENABLE_DOCKER_IN_DOCKER="y"
+        - '100-gce': # kubernetes-garbagecollector-100-gce
```

Comment:
Give that you manually tested, I don't think we need to continuously test this in every configuration. We need to continuously test in at least one configuration to ensure that it continues to work, ideally in one presubmit test and one post submit. But I don't think we need to duplicate the 2000 node test, for example.